core/domain/classroom_config_services.py

```diff
@@ -79,13 +79,25 @@ def get_classroom_from_classroom_model(
         Classroom. A classroom domain object corresponding to the given
         classroom model.
     """
+    thumbnail_data = classroom_config_domain.Image(
```

Comment:
You're calling this variable thumbnail_data ... does that mean the domain object should really be ImageData instead?

Reply:
I agree. let's update it.

Reply:
Done.